### ISSUE TITLE
Entering a large string as a get param on the events search causes a 500

### ISSUE THREAD

**Issue by rossbruniges** [mozilla/make.mozilla.org#115]
Originally reported in bugzilla, have tracked the problem down to urllib2

**Comment by rossbruniges** [comment]
So as we can see this has been merged into the dev branch - waiting on confirmation from the reporter in bugzilla if it's been suitably fixed.
